<launch>

    <arg name="project" default="lio_sam"/>
    <param name="/use_sim_time" value="true" />
    
    <!-- Parameters -->
    <rosparam file="$(find lio_sam)/config/params_ue5.yaml" command="load" />

    <!--- LOAM -->
    <include file="$(find lio_sam)/launch/include/module_loam.launch" />

    <remap from="/xianfeng_lidar" to="/points_raw" />
    <remap from="/xianfeng_imu" to="/imu_raw" />
    <remap from="/xianfeng_imu" to="/imu_correct" />

    <!--- Robot State TF -->
    <!-- 机器人描述和机器人状态话题，就是各个传感器之间位置关系 -->
    <include file="$(find lio_sam)/launch/include/module_robot_state_publisher.launch" />

    <!--- Run Navsat -->
    <!-- 就是GPS相关的话题 -->
    <include file="$(find lio_sam)/launch/include/module_navsat.launch" />

    <!--- Run Rviz-->
    <include file="$(find lio_sam)/launch/include/module_rviz.launch" />

    <!--- Run Rosbag-->
    <include file="$(find lio_sam)/launch/include/module_rosbag.launch" />

</launch>
